rust/protocol/tests/ratchet.rs

```diff
@@ -48,29 +48,29 @@ fn test_ratcheting_session_as_bob() -> Result<(), SignalProtocolError> {
 
     let alice_base_public_key = PublicKey::deserialize(&alice_base_public)?;
 
-    let alice_identity_public = IdentityKey::decode(&alice_identity_public)?;
-
     let bob_parameters = BobSignalProtocolParameters::new(
         bob_identity_key_pair,
         bob_signed_prekey_pair,
         None, // one time pre key pair
         bob_ephemeral_pair,
-        alice_identity_public,
+        IdentityKey::decode(&alice_identity_public)?,
         alice_base_public_key,
     );
 
     let bob_session = initialize_bob_session(&bob_parameters)?;
 
+    let bob_record = SessionRecord::new(bob_session);
+
     assert_eq!(
-        bob_session.local_identity_key()?,
-        *bob_identity_key_pair.identity_key()
+        hex::encode(bob_record.local_identity_key_bytes()?),
```

Comment:
I assume the `hex::encode` is to make for prettier output when the assertion fails?

Reply:
Exactly, not that it should ever fail, but trying to eyeball compare hex is a little easier than the array notation.